docs/cloud-native-security/index.rst

```diff
@@ -126,76 +122,48 @@ include: review of the customer’s runbook, summary of findings and
+* Amazon GuardDuty – **MEDIUM**
+* AWS Security Hub (Security Standards) – **HIGH**
+
+Per-event billing model
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Customers that opt for this billing model will be charged a set fee for every security event Rackspace responds to. The monthly charge for the service is calculated as the product of the number of events multiplied by the cost per event, or the minimum monthly fee, whichever is greater.
+
+The per-event fee will apply to events (SIEM-correlated findings). For example, if the native security products generate five findings that are delivered to the SIEM and, after correlation, the SIEM generates two events (one for two related findings and one for three related findings), the customer will be charged for two events only.
+
+In order to give customers some control over costs (which have a direct correlation with the amount of findings Rackspace responds to) and to help with prioritization, we will define a minimum severity level for each customer. Rackspace will only respond to findings that are at or above the defined minimum severity level. Findings with lower severity level will not be processed by Rackspace and the customer won’t be charged for those. The minimum severity level will be agreed upon between the customer and Rackspace.
+
+Initial selection of the minimum severity level for Rackspace response can either be based on a review of the native security products in the customer’s environment (if they are already enabled) or simply be set to the highest severity level (i.e. Critical/High) to start with. As part of the regular monthly account reviews, the minimum severity level will be reviewed by Rackspace and the customer. When the current setting does not generate too many findings, we can reduce the minimum severity level and start responding to findings with a lower severity. Likewise, if the current setting is generating too many findings, we can increase the minimum severity level.
 
 Additional Services
 -------------------
 
-Cloud Native Security (as the name suggests) provides managed security services
-using native AWS security products. Customers interested in improving their
-security posture over and above what is possible from the Cloud Native Security
-Service Block alone can purchase the
-`Rackspace Proactive Detection and Response <https://www.rackspace.com/managed-security-services>`_
-service block. Proactive Detection and Response includes all the capabilities
-of Cloud Native Security, plus additional 3rd party products, agent-based OS
-level threat detection, threat hunting and threat intelligence.
+Cloud Native Security provides SOC-as-a-Service for native AWS security products. Customers interested in improving their security posture over and above what is possible from the Cloud Native Security Service Block alone can purchase `add-on services <https://www.rackspace.com/security>`_ for agent-based OS level threat detection, threat hunting, and threat intelligence.
```

Comment:
Cloud-Native Security provides ...
... from the Cloud-Native Security Service Block. ...
... agent-based OS-level threat detection ...

Reply:
cannot change 1 and 2.
3 - updated